### PR TITLE
Bundle will have category

### PR DESCRIPTION
**Pull request by bmoelans**
https://vikingco.atlassian.net/browse/UWA-1360

### CODE REVIEW

File: definitions/subscriptions/balance.yaml [hunk 1]
```diff
@@ -43,6 +44,12 @@ properties:
       properties:
         descriptions:
           $ref: '#/definitions/Descriptions'
+        category:
+          type: string
+          enum:
+            - bundle
```
Comment (nielsvanoch) [inline definitions/subscriptions/balance.yaml]: Are these the best names? A SON bundle is also a bundle. Plus, the word SON is not really known.

Comment (bmoelans) [reply definitions/subscriptions/balance.yaml]: maybe
* on_net
* off_net ?

Comment (bmoelans) [reply definitions/subscriptions/balance.yaml]: as discussed IRL
* default
* super_on_net

**Comment by daneov** [comment]
Does this make `sms_super_on_net ` and `voice_super_on_net` obsolete?

**Comment by bmoelans** [comment]
@daneov we for sure will note delete since for Jim, they will have an unknown category for now.
But on the long run, I guess we could remove them (if we have versioning in UWA)

**Pull request merged**